guides/tasks/INC_ASG_BOXES.md

```diff
@@ -0,0 +1,47 @@
+## Adjusting the number of ASG boxes
+
+For example, in the publishing group:
+
+```shell
+cd dp-setup/terraform/publishing
+gpg -d develop.tfvars.asc >develop.tfvars
+```
+
+Create a feature branch.
+
+Look in file **`develop.tfvars`** at: **`asg_max_size`** and **`asg_min_size`** and increase those values by **`1`** (NOTE: they should both be the same value)
+
+Encrypt the changed file: **`develop.tfvars`**, with:
+
+```shell
+gpg -ear develop ./develop.tfvars
+```
+
+Push the changed file (**`develop.tfvars.asc`**), do a PR, get it approved.
+
+Apply the following commands:
+
+(NOTE: The first init bellow may throw out some problems, so cd into appropriate directories and attend to any needed terraform init's / apply's ...)
+
+```shell
+terraform init
+terraform workspace list
+terraform workspace select develop
+terraform plan -var-file=develop.tfvars
+terraform apply -var-file=develop.tfvars
+```
+
+This may involve cd'ing into dp-ci/terraform/s3 where the the **`plan`** and **`apply`** commands would be:
```

Comment:
dp-ci/terraform/s3 => `dp-ci/terraform/s3` for consistency perhaps?

Reply:
Done.